src/test/gcp/cloudscheduler.spec.ts

```diff
@@ -0,0 +1,104 @@
+import * as _ from "lodash";
+import { expect } from "chai";
+import * as nock from "nock";
+import * as api from "../../api";
+
+import { cloudscheduler } from "../../gcp/";
+
+const VERSION = "v1beta1";
+
+const testJob = {
+  name: "projects/test-project/locations/us-east1/jobs/test",
+  schedule: "every 5 minutes",
+  timeZone: "America/Los_Angeles",
+  httpTarget: {
+    uri: "https://afakeone.come",
+    httpMethod: "POST",
+  },
+  retryConfig: {},
+};
+
+describe("cloudscheduler", () => {
+  describe("createOrUpdateJob", () => {
+    afterEach(() => {
+      nock.cleanAll();
+    });
+
+    it("should create a job if none exists", async () => {
+      nock(api.cloudschedulerOrigin)
+        .get(`/${VERSION}/${testJob.name}`)
+        .reply(404, { context: { response: { statusCode: 404 } } });
+      const mockJobResp = { schedule: "every 5 minutes" };
```

Comment:
Shouldn't this response be the full `Job`? Not just the schedule? I know we're mocking this, but it's a bit confusing to read this, as if the API will only return the schedule and not the full Job. 

This applies everywhere else too.

Reply:
ping on this